README.md

```diff
@@ -1,3 +1,22 @@
-# Guidelines
+#GUIDELINES WIKI
+**Aspiring to be a World-Class software developer/engineer?** This is a guide where you will get engaged on the great values,
```

Comment:
Whoops, @Antoh1 didn't mean for you to copy my ... verbatim. My bad. Let's change this section to:

```
These guidelines will introduce you to the values, best software development practices and awesome opportunities the Andela community has to offer.
With them, you'll be well equipped for your journey to becoming a world-class developer.
```

Once that's in, I'll merge the PR!